pkg/search.go

```diff
@@ -96,6 +105,25 @@ type RootCause struct {
 	Index  string `json:"index"`
 }
 
+type SearchAnalytics struct {
+	Timestamp        string
```

Comment:
is string rather than some date-time?

Reply:
It is a formatted string that BigQuery can pull in and store as a `datetime` type.